public/app/features/visualization/data-hover/DataHoverView.tsx

```diff
@@ -72,8 +72,13 @@ export const DataHoverView = ({ data, rowIndex, columnIndex, sortOrder, mode, he
       });
     }
 
+    // Sanitize field by removing hovered property to fix unique display name issue
+    const sanitizedField = { ...field };
+    // Not the cleanest approach - but necessary to avoid TS error when trying to delete a required property
+    delete (sanitizedField as { hovered?: boolean }).hovered;
+
```

Comment:
instead of this, could we do something like:
```typescript
    const { hovered, ...sanitizedField } = field;

    displayValues.push({
      name: getFieldDisplayName(sanitizedField, data),
      value,
      valueString: formattedValueToString(fieldDisplay),
      highlight: hovered,
    });
```
?